### ISSUE TITLE
Headings and labels (WCAG 2.4.6)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1239]
**Request:** The intent of this Success Criterion is to help users understand what information is contained in Web pages and how that information is organized. When headings are clear and descriptive, users can find the information they seek more easily, and they can understand the relationships between different parts of the content more easily. Descriptive labels help users identify specific components within the content.

**Issues:**
1. a. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
b. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
-The heading "Title" is not meaningful in connection with the page content. In this case it should be considered to maybe not mark "Title" as a heading.

2. https://slidewiki.org/user/vivien.zschammer/recommendations 
-Heading `h1` gives no information about the subject and purpose of the area below.

**Hint:** 
#1159
Despite the German language selection, there are many English terms on the tested pages, both visible content and alternative texts:  
- https://slidewiki.org/ 
- https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
- https://slidewiki.org/search?keywords=accessibility&sort=score 
- https://slidewiki.org/user/vivien.zschammer/recommendations 
- https://slidewiki.org/addDeck 
- https://slidewiki.org/contactus 
- https://slidewiki.org/deck/127411/test/deck/127411 
- https://slidewiki.org/accessibility 
- https://slidewiki.org/user/vivien.zschammer/settings/profile
- https://slidewiki.org/user/vivien.zschammer/stats

**Comment by aoelen** [comment]
I think issue 1a and b related to the title on the slide. This can only be changed by the creator of the slide. 

Issue 2 is fixed in MR: https://github.com/slidewiki/slidewiki-platform/pull/1274

The language is still a problem, but indeed already mentioned in issue https://github.com/slidewiki/slidewiki-platform/issues/1159
